client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -5,86 +5,98 @@
-                <Container className = "profile-container">
-                    <Form className = "profile-form">
-                        <p className = "profile-title-name">Edit Profile</p>
-
-                        <Form.Group controlId="formProfilePic">
-                            <Image src={profilePic} roundedCircle height="184px" width="184px" />
-                            <Form.Control type="file" />
-                            <Form.Text className="text-muted">
-                                Upload a file from your device, at least 184px.
-                            </Form.Text>
-                        </Form.Group>
-
-                        <Row>
-                            <Col>
-                                <Form.Group controlId="formUserFirstName">
-                                    <Form.Label>First Name</Form.Label>
-                                    <Form.Control type="text" placeholder="John" />
-                                </Form.Group>
-                            </Col>
-                            
-                            <Col>
-                                <Form.Group controlId="formUserLastName">
-                                    <Form.Label>Last Name</Form.Label>
-                                    <Form.Control type="text" placeholder="Doe" />
-                                </Form.Group>
-                            </Col>
-                        </Row>
-
-                        <Form.Group controlId="formCountry">
-                                    <Form.Label>Country</Form.Label>
-                                    <Form.Control as="select" defaultValue="Australia">
-                                        <option>Australia</option>
-                                        <option>USA</option>
-                                        <option>Canada</option>
-                                    </Form.Control>
-                        </Form.Group>
-
-                        <Form.Group controlId="formEmail">
-                            <Form.Label>Email</Form.Label>
-                            <Form.Control type="email" placeholder="john.doe@gmail.com" />
-                        </Form.Group>
-
-                        <Form.Group controlId="formPhoneNumber">
-                             <Form.Label>Phone Number</Form.Label>
-                             <Form.Control type="text" placeholder="+62123456789" />
-                        </Form.Group>
+const profileUpdated = () => toast.success('Profile has been successfully updated')
+const profileDeleted = () => toast.error('Profile has not been deleted')
 
+const ProfileInfoEdit = () => {
+
+    const groupName = useSelector(state => state.user?.groupName); 
+    const orgName = useSelector(state => state.user?.orgName); 
+    const country = useSelector(state => state.user?.country); 
+    const email = useSelector(state => state.user?.email); 
+    const phoneNum = useSelector(state => state.user?.phoneNum);
```

Comment:
Is it better to combine this by just selecting the `user` data from the redux store as opposed to the individual attributes for a user? Or is there a reason to do it this way?
i.e. `const user = useSelector(state => state.user);`